Description: app/testpmd: link with virtio PMD when using shared libraries
 testpmd is often used inside a VM to test the OVS PVP scenario.
 This commit makes testpmd to link to virtio PMD when DPDK is built as
 shared libraries too.
Origin: http://dpdk.org/dev/patchwork/patch/31459/
Forwarded: yes
Author: Timothy Redaelli <tredaelli@redhat.com>
Last-Update: 2017-11-17
--- a/app/test-pmd/Makefile
+++ b/app/test-pmd/Makefile
@@ -91,6 +91,10 @@ ifeq ($(CONFIG_RTE_LIBRTE_PMD_SOFTNIC),y)
 LDLIBS += -lrte_pmd_softnic
 endif
 
+ifeq ($(CONFIG_RTE_LIBRTE_VIRTIO_PMD),y)
+LDLIBS += -lrte_pmd_virtio
+endif
+
 endif
 
 CFLAGS_cmdline.o := -D_GNU_SOURCE
